FAQ Application/README.md

```diff
@@ -0,0 +1,14 @@
+# FAQ Accordion
+
+## Description
+
+
```

Comment:
The description is blank. Also, add a "Use of this project" section and describe where can this project be used.